prisma/schema.prisma

```diff
@@ -24,6 +24,7 @@ model User {
 enum AuthType {
   PASSWORD
   GITHUB
+  LINKEDIN
```

Comment:
ok lets separate out the migration file into its own pr and push that first - maybe you did that already?